src/components/common/Footer.tsx

```diff
@@ -19,7 +19,7 @@ const Footer = ({ config }: FooterProperties) => {
       <div className='mx-auto max-w-7xl px-4 py-4 text-center'>
         {!isHomePage && <SocialMediaLink socialMedia={config.socialMedia} />}
         <p className='text-[var(--gray)]'>
-          {`© ${config.startYear && config.startYear < currentYear ? `${config.startYear} - ` : ''}${currentYear} ${config.title}`}
```

Comment:
Type error: 'config.startYear' is possibly 'null'.

That's why we need to make sure `config.startYear` before we do the comparison.